src/oauth.test.ts

```diff
@@ -1367,30 +1367,6 @@ describe.sequential("OAuth", () => {
       expect(Object.keys(responseBody)).toEqual(["error", "error_description"]);
       expect(responseBody.error).toBe("unsupported_grant_type");
     });
-
-    // Invalid request
-    it("cannot use unknown parameters", async () => {
```

Comment:
Probably just negate these tests to make them assert the positive "can pass scope" etc